build.sbt

```diff
@@ -218,6 +218,8 @@ lazy val commonSettings = Seq(
     "-Ycache-macro-class-loader:last-modified", // and macro definitions. This can lead to performance improvements.
     "-Ywarn-value-discard" // Emit warning and location for usages of deprecated APIs.
   ),
+  Compile / console / scalacOptions := (Compile / scalacOptions).value.filterNot(_ == "-Xfatal-warnings"),
```

Comment:
I guess we'd like to take the filter out at the end.

Reply:
This only applies when you run `console` - compile and test scalac settings are not changed.